collector/benchmarks/native-tls-0.1.5/perf-config.json

```diff
@@ -1,3 +1,4 @@
 {
-    "touch_file": "src/lib.rs"
+    "touch_file": "src/lib.rs",
+    "category": "primary"
```

Comment:
`native-tls-0.1.5` isn't actually a benchmark. For some reason it's in here because `tokio-webpush-simple` needs it, I think. So I'm not sure if it should even have a `perf-config.json` file at all? And perhaps we could move it elsewhere, to avoid this kind of confusion. (Not as part of this PR, though.)